### ISSUE TITLE
NumericRangeFilter should specify `lookup_type` by default

### ISSUE THREAD

**Issue by cresg820** [carltongibson/django-filter#343]
```
class MyModelList(django_filters.FilterSet):
    id = django_filters.NumericRangeFilter(label="Id range")
    class Meta:
        model = MyModel

def event_list(request):
    f = MyModelList(request.GET, queryset=MyModel.objects.all())
    return render_to_response('list.html', {'filter' : f})
```

When using with NumericRangeFilter i get this error:

```
int() argument must be a string, a bytes-like object or a number, not 'tuple'
```

and i don't understand how to fix it

----

Todo: 

* [ ] Override `__init__` adding `kwargs.setdefault('lookup_expr', 'contains')`
* [ ] Add valid lookups to Docs and docstring
* [ ] Review test cases

(CG: Added [from comment](https://github.com/carltongibson/django-filter/issues/343#issuecomment-257981681))

**Comment by rpkilby** [comment]
Hi @cresg820 - the problem should be:
```py
class MyModelList(django_filters.FilterSet):
    id = django_filters.NumericRangeFilter(label="Id range", lookup_type='range')
```

**Comment by rpkilby** [comment]
That said, specifying the `lookup_type` shouldn't be necessary for this filter. That is a bug that should be fixed.

**Comment by rpkilby** [comment]
Also, keep in mind that this is special range filter for postgres numerical range fields. See docs [here](https://django-filter.readthedocs.org/en/latest/ref/filters.html#numericrangefilter). You probably just want a standard `RangeFilter`

**Comment by rpkilby** [comment]
Hi @carltongibson - was looking over this again and realized that I completely misunderstood the purpose of the filter. `NumericRange` is the data type - this has nothing to do with `__range` lookup expressions. In fact, `range` isn't even a valid lookup on the `NumericRange` type. 

I don't think there's an appropriate default `lookup_expr` here, as there isn't a good equivalent to an `exact` lookup. The valid lookups, are `overlap`, `contains`, `contained_by`, `fully_lt`, `fully_gt`, `not_lt`, `not_gt`, `adjacent_to`, `startswith`, `endswith`, `isempty`.

Ideas:
- Make `lookup_expr` default to `None` and throw an exception if not overridden, explaining that it is required for this filter.
- Maybe move this into a postgres subpackage and add other postgres specific filters?

**Comment by carltongibson** [comment]
I think for a range `contains` is a pretty good default... I'm not too keen on a Postgres sub package  just yet...

**Comment by carltongibson** [comment]
Todo: 

* [ ] Override `__init__` adding `kwargs.setdefault('lookup_expr', 'contains')`
* [ ] Add valid lookups to Docs and docstring
* [ ] Review test cases

**Comment by rpkilby** [comment]
Hey @carltongibson - I feel dumb. 
- `exact` lookups are completely valid. The ones I found a while back are *just* the containment lookups.
- The docs do include relevant relevant information about the nature of `NumericRangeFilter` and the additional lookups. 
- Test cases are all in order. 

I don't think there's really anything to do here except clean up some of the reference docs formatting.

**Comment by carltongibson** [comment]
Closed by #540
